Dockerfile

```diff
@@ -3,31 +3,33 @@
 # Version SHA has been removed, see: https://github.com/DiamondLightSource/blueapi/issues/1053
 ARG PYTHON_VERSION=3.11
 FROM python:${PYTHON_VERSION} AS developer
```

Comment:
I switched to ubuntu and use a uv managed python, this seems to work now

Reply:
Difficult to tell, the download takes a fraction of a second on my machine, so I miss the output. I think it is downloaded each time, but I can't be sure